src/XMLSecurityDSig.php

```diff
@@ -99,54 +120,91 @@ public function __construct($prefix='ds')
         $this->sigNode = $sigdoc->documentElement;
     }
 
-    private function resetXPathObj()
```

Comment:
I agree - I only reverted the whitespace- and linebreaking-changes - making a diff with the previous Version I did not realize that the IDE not only makes this changes but also rearranged the code. 
I will go back to Rev. 56783df67a3c9d2d0dd1e82a1b7daa5033dba4de and make the needed changes again.